snuba/datasets/transactions_processor.py

```diff
@@ -79,6 +87,9 @@ def process_message(self, message, metadata=None) -> Optional[ProcessedMessage]:
             data["timestamp"],
         )
 
+        duration = processed["finish_ts"] - processed["start_ts"]
```

Comment:
Do we need to strip the signature of this since the duration column is unsigned, and this very well could end up negative?

Reply:
Does the `processed` dict have `ms` still? It looks like ms are split out on line 86 and then not factored into the duration calculation.

Reply:
I can do that, but wouldn't the result be wrong anyway ?
if we get a start date higher than finish date, whether we put 0, finish - start or abs(finish - start) all three values are wrong since the input cannot be right.

Reply:
it still has milliseconds. Line 86 extracts the milliseconds for the ms column but it does not remove them from the date object.

Reply:
Oh, I guess I don't see where the `0` is coming from on mismatched timestamps.

Reply:
with the last update now we store 0 if the duration is negative